static/app/views/organizationGroupDetails/groupMerged/mergedItem.tsx

```diff
@@ -70,42 +66,56 @@ class MergedItem extends React.Component<Props, State> {
   }
 
   handleToggle = () => {
-    const {disabled, fingerprint, event} = this.props;
+    const {fingerprint} = this.props;
+    const {latestEvent} = fingerprint;
 
-    if (disabled || this.state.busy) {
+    if (this.state.busy) {
       return;
     }
 
     // clicking anywhere in the row will toggle the checkbox
-    GroupingActions.toggleUnmerge([fingerprint, event.id]);
+    GroupingActions.toggleUnmerge([fingerprint.id, latestEvent.id]);
   };
 
   handleCheckClick() {
     // noop because of react warning about being a controlled input without `onChange`
     // we handle change via row click
   }
 
+  renderFingerprint(id: string | undefined, label?: string) {
```

Comment:
if `id` and label are `undefined` shouldn't we return `null`?

Reply:
we won't stick with this UI anyway. Choosing to merge for now

Reply:
`id` cannot be `null` but I can't figure out how to fix the types atm

Reply:
as you said.. everything will change so we can merge as it is